classes/class-wc-connect-nux.php

```diff
@@ -98,6 +98,37 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 			return $pointers;
 		}
 
+		public static function get_banner_type_to_display( $status = array() ) {
+			/* The NUX Flow:
+			- Case 1: Jetpack not connected (with TOS or no TOS accepted):
+				1. show_banner_before_connection()
+				2. connect to JP
+				3. show_banner_after_connection(), which sets the TOS acceptance in options
+			- Case 2: Jetpack connected, no TOS
+				1. show_tos_only_banner(), which accepts TOS on button click
+			- Case 3: Jetpack connected, and TOS accepted
+				This is an existing user. Do nothing.
+			*/
+			switch ( $status['jetpack_connection_status'] ) {
+				case self::JETPACK_UNINSTALLED:
+				case self::JETPACK_INSTALLED:
+				case self::JETPACK_ACTIVATED:
```

Comment:
Similarly, I think several of these constants could be renamed to something that is clearer:
JETPACK_INSTALLED -> JETPACK_NOT_ACTIVATED
JETPACK_ACTIVATED -> JETPACK_NOT_CONNECTED